layer.md

```diff
@@ -44,10 +44,45 @@ Where supported, MUST include file attributes for Additions and Modifications in
     * Group Name (`gname`) *secondary to `gid`*
 * Mode (`mode`)
 * Extended Attributes (`xattrs`)
-* Symlink reference (`linkname`)
+* Symlink reference (`linkname` + symbolic link type)
+* [Hardlink](#hardlinks) reference (`linkname`)
 
 [Sparse files](https://en.wikipedia.org/wiki/Sparse_file) SHOULD NOT be used because they lack consistent support across tar implementations.
 
+#### Hardlinks
+
+Hardlinks are a [POSIX concept](http://pubs.opengroup.org/onlinepubs/9699919799/functions/link.html) for having one or more directory entries for the same file on the same device.
+Not all filesystems support hardlinks (e.g. [FAT](https://en.wikipedia.org/wiki/File_Allocation_Table)).
+
+Hardlinks are possible with all [file types](#file-types) except `directories`.
+Non-directory files with linkcount greater than 1 have hardlinks.
+The corresponding files that share the link with the > 1 linkcount may be outside the directory that the changeset is being produced from, in which case the `linkname` is not recorded in the changeset.
+
+Hardlinks are stored in a tar archive with type of `'1'`, per the [Basic Tar Format][tar-standard].
```

Comment:
Linking to the GNU docs makes it less clear where this fits in the basket of tar standards ;).  [libarchive's tar(5)][1] attributes this typeflag to ustar (IEEE Std 1003.1-1988).  IEEE Std 1003.1-1988 has been [superseded by IEEE Std 1003.1-2008][2], which seems to have been [superseded by IEEE Std 9945-2009][3].  The IEEE versions are paywalled, so I'm not sure what's inside.  But it's probably better to mention ustar and successors here, and possibly link to the libarchive man page for more historical context, instead of linking to the GNU docs.

[1]: https://github.com/libarchive/libarchive/wiki/ManPageTar5#POSIX_ustar_Archives
[2]: http://standards.ieee.org/findstds/standard/1003.1-1988.html
[3]: http://standards.ieee.org/findstds/standard/1003.1-2008.html
[3]: https://www2.opengroup.org/ogsys/catalog/T101